src/generators/schema/webpage.php

```diff
@@ -96,13 +97,119 @@ public function add_author( $data, $post ) {
 	 * If we have an image, make it the primary image of the page.
 	 *
 	 * @param array $data WebPage schema data.
+	 *
+	 * @return array $graph The new graph with added image content.
 	 */
-	public function add_image( &$data ) {
+	public function add_images( $data ) {
+		$data = $this->add_primary_image( $data );
```

Comment:
I would move this call outside the add_images function, as it takes care of the primery image-related properties.